include/uapi/sound/sof/abi.h

```diff
@@ -28,8 +28,8 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 20
-#define SOF_ABI_PATCH 1
+#define SOF_ABI_MINOR 21
+#define SOF_ABI_PATCH 0
```

Comment:
sorry, I screwed up and should have reset this patch level earlier. I added https://github.com/thesofproject/linux/pull/3610 to fix this, you'll have to rebase when it's merged. My bad.

Reply:
@plbossart  Never mind, I will rebase this PR when #3610 is merged. Thanks.

Reply:
@plbossart  I rebase this PR, thanks.